kubernetes/structures_container.go

```diff
@@ -418,7 +418,7 @@ func flattenContainers(in []v1.Container, serviceAccountRegex string) ([]interfa
 				if err != nil {
 					return att, err
 				}
-				if nameMatchesDefaultToken {
+				if nameMatchesDefaultToken || strings.HasPrefix(m.Name, "kube-api-access") {
```

Comment:
What is this about?

Reply:
All of the Pod tests were broken and stuck in a force re-create loop because this token is added by default on EKS.